src/v/raft/consensus.h

```diff
@@ -559,6 +564,7 @@ class consensus {
     // consensus state
     model::offset _commit_index;
     model::term_id _term;
+    model::term_id _confirmed_term;
```

Comment:
Would be good to have a comment here explaining the difference between term and confirmed_term